scripts/build_gdl.sh

```diff
@@ -481,17 +494,54 @@ function pack_gdl {
         export GDL_INSTALL_DIR=`cygpath -w ${ROOT_DIR}/install`
         export GDL_VERSION=`grep -oP 'set\(VERSION "\K.+(?="\))' ${GDL_DIR}/CMakeLists.txt`
         makensis -V3 ${GDL_DIR}/scripts/deps/windows/gdlsetup.nsi
+    elif [ ${BUILD_OS} == "macOS" ]; then
+        mkdir -p "${ROOT_DIR}/package/GNU Data Language.app/Contents"
+        cd "${ROOT_DIR}/package/GNU Data Language.app/Contents"
+
+        mkdir MacOS
+        echo "#!/bin/bash" > MacOS/gdl
```

Comment:
I would recommend to use `/bin/sh` here, as this is the POSIX standard. Currently, Mac replaced the default shell with zsh, and it is a question how long bash survives there. `/bin/sh` is however always available.
```suggestion
        echo "#!/bin/sh" > MacOS/gdl
```
(as long as you don't need specific bash features, however)